### PR TITLE
feat: transfer estates

### LINKED ISSUE #308: Transfer Estate
call `safeTransfer` from `EstateRegistry`

### PR DESCRIPTION
**Pull request by nachomazzara**
Close #308

### CODE REVIEW

File: webapp/src/components/TransferAssetForm/TransferAssetForm.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.TransferAssetForm .ui.input.address-input {
+  max-width: 450px !important;
+  display: block;
+}
+
+.TransferAssetForm .ui.input.address-input input {
+  width: 100% !important;
```
Comment (NicoSantangelo) [inline webapp/src/components/TransferAssetForm/TransferAssetForm.css]: Are these `!important`s necessary?
even if they're copied from another .css file I think we can avoid them using specificity

File: webapp/src/components/EstateDetailPage/EstateDetail.js [hunk 2]
```diff
@@ -53,13 +53,16 @@ export default class EstateDetail extends React.PureComponent {
...
+                    onEditMetadata={onEditMetadata}
+                    assetId={estate.asset_id}
+                  />
                 </Grid.Column>
               )}
               {allParcels && (
                 <React.Fragment>
                   <Grid.Column
-                    width={8}
+                    width={isOwner ? 8 : 16}
```
Comment (NicoSantangelo) [inline webapp/src/components/EstateDetailPage/EstateDetail.js]: This is kind of weird. Maybe abstracting into a constant? naming floating numbers it's (almost) always a good idea

File: monitor/reducers/estateReducer.js [hunk 3]
```diff
@@ -92,17 +92,17 @@ export async function estateReducer(events, event) {
...
 
       const last_transferred_at = await new BlockTimestampService().getBlockTime(
         block_number
       )
 
       await Estate.update(
-        { owner: to.toLowerCase(), last_transferred_at },
-        { asset_id: estateId }
+        { owner: _to.toLowerCase(), last_transferred_at },
+        { asset_id: _tokenId }
```
Comment (NicoSantangelo) [inline monitor/reducers/estateReducer.js]: Do we have a companion PR on https://github.com/decentraland/land for this?

File: webapp/src/components/TransferAssetForm/TransferAssetForm.css (hunk 1 above)
Comment (nachomazzara) [reply webapp/src/components/TransferAssetForm/TransferAssetForm.css]: The `max-width` it is being used but we can remove both due other inputs don't have a `max-width`

File: monitor/reducers/estateReducer.js (hunk 3 above)
Comment (nachomazzara) [reply monitor/reducers/estateReducer.js]: No, `EstateRegistry` inherit from the [ERC721 standard](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/token/ERC721/ERC721Basic.sol#L42) which `transfer` method has that names for `args`

**Pull request merged**